### PR TITLE
Increase default cluster start timeouts and make configurable (backport)

### LINKED ISSUE #1269: KafkaContainerCluster start up sometimes times out
The `KafkaContainerCluster` start up will sometimes timeout. Container logging should be available to help troubleshoot.

Ex) 

```
[info]   java.lang.RuntimeException: java.util.concurrent.TimeoutException
[info]   at akka.kafka.testkit.internal.KafkaContainerCluster.start(KafkaContainerCluster.java:170)
[info]   at akka.kafka.testkit.internal.TestcontainersKafka$Spec.startCluster(TestcontainersKafka.scala:78)
[info]   at akka.kafka.testkit.internal.TestcontainersKafka$Spec.startCluster$(TestcontainersKafka.scala:65)
[info]   at akka.kafka.TransactionsSourceSpec.startCluster(TransactionsSourceSpec.scala:29)
[info]   at akka.kafka.testkit.internal.TestcontainersKafka$Spec.startCluster(TestcontainersKafka.scala:63)
[info]   at akka.kafka.testkit.internal.TestcontainersKafka$Spec.startCluster$(TestcontainersKafka.scala:63)
[info]   at akka.kafka.TransactionsSourceSpec.startCluster(TransactionsSourceSpec.scala:29)
[info]   at akka.kafka.testkit.scaladsl.TestcontainersKafkaPerClassLike.setUp(TestcontainersKafkaPerClassLike.scala:16)
[info]   at akka.kafka.testkit.scaladsl.TestcontainersKafkaPerClassLike.setUp$(TestcontainersKafkaPerClassLike.scala:15)
[info]   at akka.kafka.TransactionsSourceSpec.setUp(TransactionsSourceSpec.scala:29)
[info]   at akka.kafka.testkit.internal.TestFrameworkInterface$Scalatest.beforeAll(TestFrameworkInterface.scala:21)
[info]   at akka.kafka.testkit.internal.TestFrameworkInterface$Scalatest.beforeAll$(TestFrameworkInterface.scala:20)
[info]   at akka.kafka.testkit.scaladsl.ScalatestKafkaSpec.beforeAll(ScalatestKafkaSpec.scala:11)
[info]   at org.scalatest.BeforeAndAfterAll.liftedTree1$1(BeforeAndAfterAll.scala:212)
[info]   at org.scalatest.BeforeAndAfterAll.run(BeforeAndAfterAll.scala:210)
[info]   at org.scalatest.BeforeAndAfterAll.run$(BeforeAndAfterAll.scala:208)
[info]   at akka.kafka.scaladsl.SpecBase.org$scalatest$wordspec$AnyWordSpecLike$$super$run(SpecBase.scala:16)
[info]   at org.scalatest.wordspec.AnyWordSpecLike.$anonfun$run$1(AnyWordSpecLike.scala:1190)
[info]   at org.scalatest.SuperEngine.runImpl(Engine.scala:535)
[info]   at org.scalatest.wordspec.AnyWordSpecLike.run(AnyWordSpecLike.scala:1190)
[info]   at org.scalatest.wordspec.AnyWordSpecLike.run$(AnyWordSpecLike.scala:1188)
[info]   at akka.kafka.scaladsl.SpecBase.run(SpecBase.scala:16)
[info]   at org.scalatest.tools.Framework.org$scalatest$tools$Framework$$runSuite(Framework.scala:318)
[info]   at org.scalatest.tools.Framework$ScalaTestTask.execute(Framework.scala:513)
[info]   at sbt.ForkMain$Run.lambda$runTest$1(ForkMain.java:413)
[info]   at java.util.concurrent.FutureTask.run(FutureTask.java:266)
[info]   at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
[info]   at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
[info]   at java.lang.Thread.run(Thread.java:748)
[info]   Cause: java.util.concurrent.TimeoutException:
[info]   at java.util.concurrent.CompletableFuture.timedGet(CompletableFuture.java:1784)
[info]   at java.util.concurrent.CompletableFuture.get(CompletableFuture.java:1928)
[info]   at akka.kafka.testkit.internal.KafkaContainerCluster.start(KafkaContainerCluster.java:145)
[info]   at akka.kafka.testkit.internal.TestcontainersKafka$Spec.startCluster(TestcontainersKafka.scala:78)
[info]   at akka.kafka.testkit.internal.TestcontainersKafka$Spec.startCluster$(TestcontainersKafka.scala:65)
[info]   at akka.kafka.TransactionsSourceSpec.startCluster(TransactionsSourceSpec.scala:29)
[info]   at akka.kafka.testkit.internal.TestcontainersKafka$Spec.startCluster(TestcontainersKafka.scala:63)
[info]   at akka.kafka.testkit.internal.TestcontainersKafka$Spec.startCluster$(TestcontainersKafka.scala:63)
[info]   at akka.kafka.TransactionsSourceSpec.startCluster(TransactionsSourceSpec.scala:29)
[info]   at akka.kafka.testkit.scaladsl.TestcontainersKafkaPerClassLike.setUp(TestcontainersKafkaPerClassLike.scala:16)
[info]   at akka.kafka.testkit.scaladsl.TestcontainersKafkaPerClassLike.setUp$(TestcontainersKafkaPerClassLike.scala:15)
[info]   at akka.kafka.TransactionsSourceSpec.setUp(TransactionsSourceSpec.scala:29)
[info]   at akka.kafka.testkit.internal.TestFrameworkInterface$Scalatest.beforeAll(TestFrameworkInterface.scala:21)
[info]   at akka.kafka.testkit.internal.TestFrameworkInterface$Scalatest.beforeAll$(TestFrameworkInterface.scala:20)
[info]   at akka.kafka.testkit.scaladsl.ScalatestKafkaSpec.beforeAll(ScalatestKafkaSpec.scala:11)
[info]   at org.scalatest.BeforeAndAfterAll.liftedTree1$1(BeforeAndAfterAll.scala:212)
[info]   at org.scalatest.BeforeAndAfterAll.run(BeforeAndAfterAll.scala:210)
[info]   at org.scalatest.BeforeAndAfterAll.run$(BeforeAndAfterAll.scala:208)
[info]   at akka.kafka.scaladsl.SpecBase.org$scalatest$wordspec$AnyWordSpecLike$$super$run(SpecBase.scala:16)
[info]   at org.scalatest.wordspec.AnyWordSpecLike.$anonfun$run$1(AnyWordSpecLike.scala:1190)
[info]   at org.scalatest.SuperEngine.runImpl(Engine.scala:535)
[info]   at org.scalatest.wordspec.AnyWordSpecLike.run(AnyWordSpecLike.scala:1190)
[info]   at org.scalatest.wordspec.AnyWordSpecLike.run$(AnyWordSpecLike.scala:1188)
[info]   at akka.kafka.scaladsl.SpecBase.run(SpecBase.scala:16)
[info]   at org.scalatest.tools.Framework.org$scalatest$tools$Framework$$runSuite(Framework.scala:318)
[info]   at org.scalatest.tools.Framework$ScalaTestTask.execute(Framework.scala:513)
[info]   at sbt.ForkMain$Run.lambda$runTest$1(ForkMain.java:413)
[info]   at java.util.concurrent.FutureTask.run(FutureTask.java:266)
[info]   at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
[info]   at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
[info]   at java.lang.Thread.run(Thread.java:748)
```

### PR DESCRIPTION
**Pull request by seglo**
Fixes #1269

Increase default cluster start timeouts and make configurable.

Based on #1310

### CODE REVIEW

File: testkit/src/main/scala/akka/kafka/testkit/KafkaTestkitTestcontainersSettings.scala [hunk 1]
```diff
@@ -219,7 +267,9 @@ final class KafkaTestkitTestcontainersSettings private (
     s"numBrokers=$numBrokers," +
     s"internalTopicsReplicationFactor=$internalTopicsReplicationFactor," +
     s"useSchemaRegistry=$useSchemaRegistry," +
-    s"containerLogging=$containerLogging)"
+    s"containerLogging=$containerLogging, +" +
+    s"clusterStartTimeout=$clusterStartTimeout," +
+    s"readinessCheckTimeout=$readinessCheckTimeout)"
```
Comment (ennru) [inline testkit/src/main/scala/akka/kafka/testkit/KafkaTestkitTestcontainersSettings.scala:272]: Possibly add `toCoarsest`

File: testkit/src/main/java/akka/kafka/testkit/internal/AlpakkaKafkaContainer.java [hunk 2]
```diff
@@ -172,7 +172,9 @@ public String getJmxServiceUrl() {
   @Override
   protected void doStart() {
     withCommand(
-        "sh", "-c", "while [ ! -f " + STARTER_SCRIPT + " ]; do sleep 0.1; done; " + STARTER_SCRIPT);
+        "sh",
+        "-c",
+        "while [ ! -f " + START_STOP_SCRIPT + " ]; do sleep 0.1; done; " + START_STOP_SCRIPT);
```
Comment (ennru) [inline testkit/src/main/java/akka/kafka/testkit/internal/AlpakkaKafkaContainer.java:177]: So this other script is included already, right?

**Review by ennru — APPROVED** [review]
LGTM.

File: testkit/src/main/java/akka/kafka/testkit/internal/AlpakkaKafkaContainer.java (hunk 2 above)
Comment (seglo) [reply testkit/src/main/java/akka/kafka/testkit/internal/AlpakkaKafkaContainer.java:177]: Yes, it just wasn't referenced in this start command.

File: testkit/src/main/scala/akka/kafka/testkit/KafkaTestkitTestcontainersSettings.scala (hunk 1 above)
Comment (seglo) [reply testkit/src/main/scala/akka/kafka/testkit/KafkaTestkitTestcontainersSettings.scala:272]: :+1:

**Pull request merged**